front/login.php

```diff
@@ -86,7 +90,17 @@
 
 
 // now we can continue with the process...
-if ($auth->login($login, $password, (isset($_REQUEST["noAUTO"]) ? $_REQUEST["noAUTO"] : false), $remember, $login_auth)) {
+if (isset($_REQUEST['cancel'])) {
```

Comment:
should we use more precise keys like `totp_cancel`, `totp_code`, `backup_code` when using POST variables between redirections ?